RELATED-PROJECTS.md

```diff
@@ -0,0 +1,49 @@
+# Other projects related to Jamulus
```

Comment:
Are you sure you want to add this to the main repo? Maybe it would be better to have it as a post in the community KB on the website.

Reply:
Maybe. I just didn't want to completely remove information from the repo, so I created this document as somewhere to put it. If we moved it to the web site and removed it from here, people cloning the repo might not know the information is available to be found elsewhere. I don't always find things easy to locate on the website.

Reply:
>  I don't always find things easy to locate on the website.

Some advanced stuff is quite hidden… probably worth re-organising the site?

Personally, I‘d prefer to have documentation on the website, but it’s up to you where to put it. 

Asking for another opinion:
@gilgongo should we move this document to the website?

Reply:
Might be nice to have it them on the website. Can't immediately think where though - perhaps as page linked out from the client and server manuals?

Reply:
Community KB?

Reply:
I suppose since these projects are a) subject to change out of our control and b) not core core information about Jamulus, then they would be a candidate for the KB, yes.